src/test/java/ArrayMinHeapTest.java

```diff
@@ -25,19 +25,113 @@ void size_empty_returnsZero() {
                     .isEqualTo(0);
         }
 
+        @Test
         void isEmpty_empty_returnsTrue() {
             assertThat(classUnderTest.isEmpty())
                     .isTrue();
         }
 
+        @Test
+        void add_empty_returnsTrue() {
+            assertThat(classUnderTest.add(10))
+                    .isTrue();
+        }
+
+        @Test
+        void add_empty_updatesHeap() {
+            classUnderTest.add(10);
+            assertThat(classUnderTest)
+                    .isNotEqualTo(preState);
+        }
```

Comment:
would it be _better_ to test that it is not equal to an new & empty queue under the nested `Singleton` test class?

Reply:
I moreso meant asserting `thing.add(x) == true`. Asserts like this are also testing `equals`. More of a workflow test